frontend/app/utils/ReferralData.scala

```diff
@@ -6,15 +6,16 @@ case class ReferralData(campaignCode: Option[String], url: Option[String], pagev
 
 object ReferralData {
 
-  val CampaignCodeKey = "mem_campaign_code"
+  val CampaignCodeKey =  "mem_campaign_code"
   val UrlKey = "gu_mem_ref_url"
   val PageviewIdKey = "gu_refpvid"
 
   def makeCookies(implicit request: RequestHeader): Seq[Cookie] = {
     val refUrl = request.headers.get("referer").map(Cookie(ReferralData.UrlKey, _))
     val refPvid = request.getQueryString("REFPVID").map(Cookie(ReferralData.PageviewIdKey, _))
+    val campaignCode = request.getQueryString("INTCMP").map(Cookie(ReferralData.CampaignCodeKey, _))
```

Comment:
True! Just looked and there isn't any reference to CMP anywhere in the membership frontend at all. Do you think we should just store CMP under INTCMP, or should we pass it through as it's own field and create a new collum in the membership_acquisition table? Maybe, seeing as on our end we treat it the same and I've never seen an instance where CMP and INTCMP is set, we could do the former? So look first for INTCMP, then failing that, look for CMP?

Good spot, I'm surprised this hasn't been spotted before!